src/pages/Scholarships.jsx

```diff
@@ -20,10 +20,11 @@ function ScholarshipsPage() {
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
+  const [sortList, setSortList] = useState('deadline');
 
   useEffect(() => {
     // TODO: Create cancellable promises
-    Scholarships.list({ sortField: 'deadline' })
+    Scholarships.list({ sortField: `${sortList}` })
```

Comment:
`Scholarships.list` inherits from `FirestoreCollection.list`, whose options are `sortDir` of type [`SortDirection`](https://github.com/beyondhb1079/s4us/blob/main/src/models/base/FirestoreCollection.ts#L5) and `sortField`, which is the field to sort by:
https://github.com/beyondhb1079/s4us/blob/main/src/models/base/FirestoreCollection.ts#L30-L33

You should be setting `sortField` and `sortDir` where it makes sense. `deadlineReverse` is not a valid field and sorting by a field doesn't mean you'll get it in the low -> high or high -> low direction necessary if `sortDir` isn't passed in.

Reply:
Punted for #342